CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [[NEXT]](https://github.com/iExecBlockchainComputing/tee-worker-post-compute/releases/tag/vNEXT) 2024
 
+### New Features
```

Comment:
I think I would prefer to see this in the **Quality** section. This is not really a new feature as we are removing Gramine support. What do you thins ?

Reply:
In the CHANGELOG, the use of Gramine variants is a feature we wish to remove. Should the removal of a feature be considered a quality improvement rather than a new feature?

Reply:
After discussion, we will introduce a new **Deprecated section**, placed before **Dependency Upgrades**